rules/windows/file/file_event/file_event_win_webshell_creation_detect.yml

```diff
@@ -27,7 +35,13 @@ detection:
             - '\htdocs\'
             - '\html\'
     selection_ext2:
-        TargetFilename|contains: '.ph'
+        TargetFilename|contains: 
+            - '.ph'
```

Comment:
Hi,
`TargetFilename|contains: '.ph'` will match ".ph",".php",".php3",...,".php7",....

You can read more information here  https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#currently-available-modifiers

Reply:
Ah, thanks, please kindly tell me how can i remove these parts on this pull request?